app/models/item.rb

```diff
@@ -0,0 +1,4 @@
+class Item < ActiveRecord::Base
+  belongs_to :user
```

Comment:
`belongs_to :user` ought to be removed for `Item`. The Item-User relationship happens through List